### PR TITLE
Improve documentation on plugin usage

### LINKED ISSUE #398: Docs usage page title/tagline examples
The `|`'s in [this section](https://github.com/jekyll/jekyll-seo-tag/blame/master/docs/usage.md#L5:L6) of the docs/usage page need to be escaped. The lines are getting rendered as tables.

### PR DESCRIPTION
**Pull request by ashmaroli**
Closes #398

### CODE REVIEW

**Comment by emma-sax4** [comment]
@ashmaroli Just in case you're still working on this, I noticed that your Travis tests are failing. Check out this: https://github.com/jekyll/jekyll-seo-tag/issues/394.

File: docs/usage.md [hunk 1]
```diff
@@ -31,9 +37,12 @@ The SEO tag will respect any of the following if included in your site's `_confi
     publisher: 1234
     admins: 1234
   ```
-
-* `logo` - URL to a site-wide logo (e.g., `/assets/your-company-logo.png`) - If you would like the "publisher" property to be present, you must add this field to your site's configuration, during the validation of the structured data by Google web master tools, if the `logo` field is not validated, you will find errors inherent to the publisher in the [structured datas test](https://search.google.com/structured-data/testing-tool/u/0/)
-* `social` - For [specifying social profiles](https://developers.google.com/structured-data/customize/social-profiles). The following properties are available:
+* `logo` - URL to a site-wide logo (e.g., `/assets/your-company-logo.png`) - If you would like the "publisher" property
+to be present, you must add this field to your site's configuration, during the validation of the structured data by
+Google web master tools, if the `logo` field is not validated, you will find errors inherent to the publisher in the
```
Comment (yg) [inline docs/usage.md:42]: Brand Guidelines!

```suggestion
Google Webmaster tools, if the `logo` field is not validated, you will find errors inherent to the publisher in the
```

File: docs/usage.md [hunk 2]
```diff
@@ -31,9 +37,12 @@ The SEO tag will respect any of the following if included in your site's `_confi
...
+[structured datas test](https://search.google.com/structured-data/testing-tool/u/0/)
```
Comment (yg) [inline docs/usage.md:43]: `datas` to `data`

```suggestion
[structured data test](https://search.google.com/structured-data/testing-tool/u/0/)
```

**Comment by DirtyF** [comment]
@jekyll: merge +docs

**Pull request merged**